tests/e2e/default/db/initial-replication.wdio-spec.js

```diff
@@ -155,6 +155,7 @@ describe('initial-replication', () => {
     setTimeout(() => browser.refresh(), 3000);
     setTimeout(() => browser.refresh(), 5000);
 
+    await browser.pause(5000);
```

Comment:
Is there a way to avoid waiting for 5 seconds?

Reply:
No. This is only because we have all those browser refresh timeouts above. we're just waiting for them to happen.

Reply:
This is just to stabilize this test.

Reply:
Ok, I think it's better to add a comment to explain it